### PR TITLE
Setup.py cleanup

### LINKED ISSUE #32: Update setup.py information
Update author and author_email to be tied to an individual.  Add maintainer as "Harvard University TLT" and maintainer email as "tlt_opensource@g.harvard.edu"

### PR DESCRIPTION
**Pull request by bermudezjd**
Update url to point to correct repo.  Use a working author email
address.  Remove Sphinx requirement for install.

Version update is dependent on #70 

Resolves #32 

@eparker71 
@elliottyates 
@rascalking 
@cmurtaugh

### CODE REVIEW

**Comment by cmurtaugh** [comment]
:+1:

File: setup.py [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 
 setup(
     name='canvas_python_sdk',
     version=version,
-    description='A python sdk for the canvas LMS api',
+    description='A python SDK for Instructure\'s Canvas LMS API',
     author='Harvard University',
-    author_email='some_email_tbd@Harvard.edu',
-    url='https://github.com/Harvard-University-iCommons/canvas_python_sdk',
+    author_email='tlt_opensource@g.harvard.edu',
```
Comment (elliottyates) [inline setup.py]: Cool. Who's monitoring this email?

**Comment by elliottyates** [comment]
:+1:

File: setup.py (hunk 1 above)
Comment (bermudezjd) [inline setup.py]: All TLT scrum team developers are now!

File: setup.py [hunk 2]
```diff
@@ -29,7 +31,6 @@
     zip_safe=False,
     install_requires=[
         'requests',
-        'sphinx>=1.2.0',
```
Comment (rascalking) [inline setup.py]: Any reason not to make this a `docs` extra?

https://pythonhosted.org/setuptools/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

**Comment by rascalking** [comment]
:+1:

File: setup.py (hunk 2 above)
Comment (bermudezjd) [inline setup.py]: OK, that makes sense.  I'll add it as an `extras_require`.

**Comment by bermudezjd** [comment]
Leaving version as same from #70 .

**Pull request merged**